tools/cockpit.spec

```diff
@@ -327,7 +327,7 @@ The Cockpit components for managing software updates for ostree based systems.
 
 %package pcp
 Summary: Cockpit PCP integration
-Requires: %{name}-bridge = %{version}-%{release}
+Requires: %{name}-bridge > 134
```

Comment:
The bridge that is declared in the same manifest.json file demands it. So I'd say yes, and bumped.